tests/system/test_root_metronome.py

```diff
@@ -124,8 +124,9 @@ def test_disable_schedule_recovery_from_master_bounce():
         job_schedule['enabled'] = False
         client.update_schedule(job_id, 'nightly', job_schedule)
 
-        # # bounce master
-        shakedown.restart_master_node()
+        # bounce master
+        metronome_leader = shakedown.master_leader_ip()
```

Comment:
unfortunately mesos leader != metronome leader. This does not have to be true. This is why I linked the ticket to the other ticket that is adding metronome leader info into the API...

Reply:
ah... that is what you mean :) .  great pt!   good eye!  and thanks.